flume-hdfs-sink/src/main/java/com/wipro/ats/bdre/flume/sink/hdfs/AbstractHDFSWriter.java

```diff
@@ -93,11 +93,11 @@ public boolean isUnderReplicated() {
       }
       return numBlocks < desiredBlocks;
     } catch (IllegalAccessException e) {
-      logger.error("Unexpected error while checking replication factor", e);
+      LOGGER.error("Unexpected error while checking replication factor", e);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Unexpected error while checking replication factor" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1192)